### PR TITLE
Null Pointer Exception on Build when no operationId is provided

### LINKED ISSUE #151: Null Pointer Exception on Build - General Logging and Error Handling
I started checking out the multiapi-maven-plugin and I ran it with the following example from the OpenAPI Studio: 

``` 
asyncapi: '2.5.0'
info:
  title: Account Service
  version: 1.0.0
  description: This service is in charge of processing user signups
channels:
  user/signedup:
    subscribe:
      message:
        $ref: '#/components/messages/UserSignedUp'
components:
  messages:
    UserSignedUp:
      payload:
        type: object
        properties:
          displayName:
            type: string
            description: Name of the user
          email:
            type: string
            format: email
            description: Email of the user
```

The maven build fails with a null pointer exception, because (as I found out after looking into the code), there is no "operationId" parameter present in the channel item. 

Expected Behaviour would be, that the generator either skips the operation with a warning or creates a default operationId (e.g. user_signedup). While looking through the code I notices, that there seems to be no error handling or logging in place. 

Exceptions are caught and handled with printstacktrace like in net.coru.api.generator.plugin.asyncapi.AsyncApiGenerator.processFileSpec(List<SpecFile>):
```      ...
      } catch (final TemplateException | IOException e) {
        e.printStackTrace();
      }
```
If this is of general interest, I would like to work on the error handling and logging by introducing a LoggingInterface in the multiapi-engine through which info, error and debug messages can be communicated to the calling plugin. In the maven plugin we would use the maven logger to implement this interface, in the gradle plugin the gradle logger.

### PR DESCRIPTION
**Pull request by davidgayoso**
Fixes #151.

### CODE REVIEW

File: multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/exception/InvalidAPIException.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.sngular.api.generator.plugin.asyncapi.exception;
+
```
Comment (5uso-sng) [inline multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/exception/InvalidAPIException.java:2]: This new file is missing its license notice

**Review by 5uso-sng — CHANGES_REQUESTED** [review]
The change seems ok, but make sure to update the version numbers

**Review by 5uso-sng — DISMISSED** [review]

File: multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/exception/InvalidAPIException.java
Comment (jemacineiras) [inline multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/exception/InvalidAPIException.java:1]: What happen with the Openapi side.. it should have the same behaviour always

**Review by jemacineiras — CHANGES_REQUESTED** [review]

File: multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/exception/InvalidAPIException.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  * License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.sngular.api.generator.plugin.openapi.exception;
+
+public class InvalidAPIException extends RuntimeException {
+
+  private static final String ERROR_MESSAGE = "OpenApi -> There is at least one operation without operationId";
```
Comment (5uso-sng) [inline multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/exception/InvalidAPIException.java:11]: Does it make sense to have this class duplicated in both openapi and asyncapi? We have a package for common exceptions

**Review by 5uso-sng — APPROVED** [review]

**Pull request merged**